v1/firebase/adapter.js

```diff
@@ -40,7 +40,18 @@ function apiGetTrack(trackId) {
 
 function apiGetChannel(channelId) {
 	return apiGet(`channels/${channelId}`).then(snapshot => {
-		return serializeChannel(snapshot.val(), channelId);
+		var channel = serializeChannel(snapshot.val(), channelId);
+
+		// Replace `images` with a single `thumbnail` URL.
+		var images = Object.keys(channel.images);
+		var lastImage = images[images.length - 1];
```

Comment:
The old images are not needed, we should in fact refactor the radio4000 client code.
Maybe a channel should have `image: belongTo` instead of `hasMany`?
Of course we would then need to clean the database filtering them by `image.created` for each chanel.
But that way it would sove this problem.

That also raises the following question:
do we need an image model in r4?